src/lsp4clj/server.clj

```diff
@@ -247,7 +247,11 @@
      :pending-requests* (atom {})
      :join (promise)}))
 
+#_{:clj-kondo/ignore [:clojure-lsp/unused-public-var]}
 (defn stdio-server
+  "DEPRECATED: Will be removed in a future release. Prefer
+  lsp4clj.io-server/server or lsp4clj.io-server/stdio-server."
+  {:deprecated true}
```

Comment:
I'm not sure what is the standard here, but usually I add a string to the value fo the deprecated keyword like:
```suggestion
  {:deprecated "Use lsp4clj.io-server/server instead"}
```